src/main/java/core/basesyntax/WorkWithFile.java

```diff
@@ -1,8 +1,53 @@
 package core.basesyntax;
 
+import java.io.BufferedReader;
+import java.io.BufferedWriter;
+import java.io.FileReader;
+import java.io.FileWriter;
+import java.io.IOException;
+
 public class WorkWithFile {
+    private static final String SEPARATOR_SYMBOL = ",";
+    private static final int OPERATION_INDEX = 0;
+    private static final int SUM_INDEX = 1;
+    private static final String SUPPLY_STRING = "supply";
+    private static final String BUY_STRING = "buy";
 
     public void getStatistic(String fromFileName, String toFileName) {
+        writeToFile(createReport(fromFileName), toFileName);
```

Comment:
```suggestion
        String dataFromFile = readFromFile(fromFileName);
        String report = createReport(dataFromFile);
        writeToFile(report, toFileName);
```
lets divide logic
read first, then create report - divide functionality